content/_index.html

```diff
@@ -1,15 +1,15 @@
 ---
 title: "Epinio"
-description: "From App to URL in one step"
+description: "The Application Development Engine for Kubernetes"
```

Comment:
Is it an "application development" or an "application deployment" engine ? 🤔

Reply:
I like development engine as it focuses on the process instead of the product. The tooling just enables the process.